### ISSUE TITLE
Several issues

### ISSUE THREAD

**Issue by BarbzYHOOL** [bstillman/mariadb-backup#40]
I cloned the repo, and tried to fill the conf but not much is explained. I don't know what backuphist means for example. (EDIT : ok I now know what it is but the explanation is confusing, you should change all names to be similar, and no mdbutil because it comes from nowhere)

I'm on debian 8 with percona 5.7

1) When I had a password with special characters, it didn't work at all. I know with xtrabackup I had to put it in ' ' but here it didn't work

2) No way to hide the password ? forced to put it in the conf ?

3) It didn't work, it kept saying backup failed, without any explanation. I had no encryption key set though (but didn't enable encryption either)
So I tried to run the command that was shown (without bgbackup) and it started to make a full backup but displayed a lot of weird characters like you find in raw text or binaries while the backup was executing (4▮�Z└[└\└]└Eb⎺┼ B┌▒de V☃┼d☃c▒├⎺⎼)
I don't know what it means, if it's going to be good or not, it's still running atm

4) Note that I never used innobackupex, because it's deprecated (and because i'm new), only used xtrabackup command before.
Why it's not using xtrabackup command though?

Atm I can't use bgbackup :(

**Comment by BarbzYHOOL** [comment]
Alright, the config is confusing.

I found what was the problem by looking at each log (thanks for the logs :P)
Problem was about the mysql socket, I should have just commented it out and not touch it (like in default setting)

Conf ideas : 

- About the scheduling, are we supposed to run this script with a cronjob ? I don't understand the fullbackday option (as a newbie on percona xtrabackup)
- Everything called "MariaDB" should be "MySQL / MariaDB / Percona Server / etc"
- backuphist should be explained clearly
- backuphistschema=mdbutil ?? why not backuphistschema=backups or something more obvious ?
- Basically the conf needs some separators for each part imo

**Comment by jdanilson** [comment]
I concur with comments above.  Why is the schema mdbutil and not something like bgbackup_history or at least something common to bgbackup.  Otherwise it's a pretty cool wrapper.
